subtrack.Tests/SubscriptionCalculatorTests/GetSubscriptionListByMonthTests.cs

```diff
@@ -50,25 +53,57 @@ public void GetSubscriptionListByMonth_MonthProvided_ShouldNotReturn_PaidSubscri
             IList<Subscription> subscriptions = CreateSubscriptions();
 
             // Act
-            var result = _sut.GetSubscriptionListByMonth(subscriptions, DateTime.Now);
+            var result = _sut.GetSubscriptionListByMonth(subscriptions, today).ToList();
 
             // Assert
-            Assert.DoesNotContain(subscriptions[2], result);
-            Assert.DoesNotContain(subscriptions[3], result);
+            Assert.DoesNotContain(subscriptions[2], result, _subscriptionComparer);
+            Assert.DoesNotContain(subscriptions[3], result, _subscriptionComparer);
         }
 
-        public static IList<Subscription> CreateSubscriptions()
+        [Fact]
+        public void GetSubscriptionListByMonth_MonthProvided_ShouldReturnWeeklySubscriptionsMultipleTimes()
         {
-            DateTime today = DateTime.Today;
+            // Arrange
+            IList<Subscription> subscriptions = CreateSubscriptions();
+
+            // Act
+            var result = _sut.GetSubscriptionListByMonth(subscriptions, today.AddMonths(2)).ToList();
+
+            // Assert
+            Assert.True(result.Count(s => s.Name == subscriptions[5].Name) == 4);
```

Comment:
place the magic numbers inside variables, can be difficult to figure out a good name but since this is a test the name can be how long you want